src/amo/components/DownloadFirefoxButton/index.js

```diff
@@ -0,0 +1,57 @@
+/* @flow */
+import classNames from 'classnames';
+import React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import translate from 'core/i18n/translate';
+import type { UserAgentInfoType } from 'core/reducers/api';
+import Button from 'ui/components/Button';
+
+
+type PropTypes = {
+  className?: 'string',
+  i18n: Object,
+  userAgentInfo: UserAgentInfoType,
+}
+
+export const DownloadFirefoxButtonBase = ({
+  className,
+  i18n,
+  userAgentInfo,
+// eslint-disable-next-line indent
+}: PropTypes = {}) => {
+  if (userAgentInfo.browser.name === 'Firefox') {
+    return null;
+  }
+
+  return (
+    <Button
+      className={classNames(
+        'DownloadFirefoxButton',
+        'Button',
+        'Button--confirm',
+        'Button--small',
+        className,
+      )}
+      href="http://mozilla.org/"
```

Comment:
> clicking on this button links to http://localhost:3000/en-US/firefox/http:/mozilla.org/ in Chrome for some reason

This is still broken for me.

Reply:
Shoot, I forgot this one, thanks!

Reply:
Turns out the button component was a bit iffy; fixed it up and added some new tests.